src/WorkItems/WorkItems.csproj

```diff
@@ -28,10 +28,11 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.2" />
     <PackageReference Include="Microsoft.Extensions.Logging.ApplicationInsights" Version="2.13.1" />
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="3.1.2" />
-    <PackageReference Include="Microsoft.Json.Schema" Version="1.1.4" />
-    <PackageReference Include="Microsoft.Json.Schema.Validation" Version="1.1.4" />
+    <PackageReference Include="Microsoft.Json.Schema" Version="1.1.5" />
+    <PackageReference Include="Microsoft.Json.Schema.Validation" Version="1.1.5" />
     <PackageReference Include="Microsoft.TeamFoundationServer.Client" Version="16.153.0" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
><PackageReference Include="Newtonsoft.Json" Version="13.0.1" /> [](http://example.com/codeflow?start=4&length=63)

this line looks new in this .csproj, is it needed

Reply:
Yes, this is needed. Otherwise, TFS will default to an older version of Newtonsoft.

Reply:
TFS --- do you mean this dll? I see it defaults to 10.0.0

![image](https://user-images.githubusercontent.com/81775155/179327902-246deadb-1467-4d51-a432-f059bddb0e24.png)

Reply:
I tried, I think this line is not need, the reason is 
Microsoft.Json.Schema 1.1.4 only needs Newtonsoft.Json (>= 9.0.1);
Microsoft.TeamFoundationServer.Client 16.153.0 needs 
Newtonsoft.Json (>= 10.0.3);
so the result is 10.0.3;
now this project already using Microsoft.Json.Schema 1.1.5 needs Newtonsoft.Json (>= 13.0.1)
so we will get 13.0.1

Reply:
Eddy experimented on this one both ways and determined that it is needed at this time.